src/adam_core/orbits/query/horizons.py

```diff
@@ -208,19 +212,38 @@ def query_horizons_ephemeris(
     """
     dfs = []
     for observatory_code, observers_i in observers.iterate_codes():
-        ephemeris = _get_horizons_ephemeris(
+        _ephemeris = _get_horizons_ephemeris(
             object_ids,
             observers_i.coordinates.time,
             observatory_code,
         )
-        dfs.append(ephemeris)
+        dfs.append(_ephemeris)
 
-    ephemeris = pd.concat(dfs, ignore_index=True)
-    ephemeris.sort_values(
+    dfs = pd.concat(dfs, ignore_index=True)
+    dfs.sort_values(
         by=["orbit_id", "datetime_jd", "observatory_code"],
         inplace=True,
         ignore_index=True,
     )
+
+    # Horizons produces UTC but we use tdb everywhere
+    epochs = Timestamp.from_jd(pa.array(dfs["datetime_jd"]), scale="utc").rescale("tdb")
```

Comment:
When we use ephemeris (like from the propagator class), we typically compare them to observations which are more likely to be in UTC so I think it makes sense to leave the ephemeris in UTC and have the user rescale downstream if needed. That said though, we haven't really used these specific functions for anything but testing so maybe its not as much of an issue.

Reply:
Maybe it makes more sense for our generic ephemeris to return in UTC then.

Reply:
Updated this to return UTC (as well as our generic ephemeris method).